packages/client/wallets/aa/src/blockchain/BlockchainNetworks.ts

```diff
@@ -32,197 +40,151 @@ Chains not supported yet due fireblocks or zerodev doesn't supported
+        ["zkatana", "ETH_ZKEVM_TEST"],
     ]).get(chain)!;
 }
-
 export function getBlockchainByChainId(chain: number) {
-    const chainIdMap = new Map<number, Blockchain>([
-        [ETHEREUM_CHAIN_ID, Blockchain.ETHEREUM],
-        [POLYGON_CHAIN_ID, Blockchain.POLYGON],
-        [BSC_CHAIN_ID, Blockchain.BSC],
-        [OPTIMISM_CHAIN_ID, Blockchain.OPTIMISM],
-        [ARBITRUM_CHAIN_ID, Blockchain.ARBITRUM],
-        [GOERLI_CHAIN_ID, Blockchain.GOERLI],
-        [SEPOLIA_CHAIN_ID, Blockchain.SEPOLIA],
-        [MUMBAI_CHAIN_ID, Blockchain.MUMBAI],
-        [ZKATANA_CHAIN_ID, Blockchain.ZKATANA],
+    const chainIdMap = new Map<number, BlockchainIncludingTestnet>([
+        [ETHEREUM_CHAIN_ID, "ethereum"],
+        [POLYGON_CHAIN_ID, "polygon"],
+        [BSC_CHAIN_ID, "bsc"],
+        [OPTIMISM_CHAIN_ID, "optimism"],
+        [ARBITRUM_CHAIN_ID, "arbitrum"],
+        [SEPOLIA_CHAIN_ID, "ethereum-sepolia"],
+        [MUMBAI_CHAIN_ID, "mumbai"],
+        [ZKATANA_CHAIN_ID, "zkatana"],
     ]);
 
     return chainIdMap.get(chain);
 }
 
-export function getChainIdByBlockchain(chain: Blockchain) {
+export function getChainIdByBlockchain(chain: BlockchainIncludingTestnet) {
     return new Map([
-        [Blockchain.ETHEREUM, ETHEREUM_CHAIN_ID],
-        [Blockchain.POLYGON, POLYGON_CHAIN_ID],
-        [Blockchain.BSC, BSC_CHAIN_ID],
-        [Blockchain.OPTIMISM, OPTIMISM_CHAIN_ID],
-        [Blockchain.ARBITRUM, ARBITRUM_CHAIN_ID],
-        [Blockchain.GOERLI, GOERLI_CHAIN_ID],
-        [Blockchain.SEPOLIA, SEPOLIA_CHAIN_ID],
-        [Blockchain.MUMBAI, MUMBAI_CHAIN_ID],
-        [Blockchain.ZKATANA, ZKATANA_CHAIN_ID],
+        ["ethereum", ETHEREUM_CHAIN_ID],
+        ["polygon", POLYGON_CHAIN_ID],
+        ["bsc", BSC_CHAIN_ID],
+        ["optimism", OPTIMISM_CHAIN_ID],
+        ["arbitrum", ARBITRUM_CHAIN_ID],
+        ["goerli", GOERLI_CHAIN_ID],
+        ["ethereum-sepolia", SEPOLIA_CHAIN_ID],
+        ["mumbai", MUMBAI_CHAIN_ID],
+        ["zkatana", ZKATANA_CHAIN_ID],
     ]).get(chain)!;
 }
 
-export function getUrlProviderByBlockchain(chain: Blockchain) {
-    return new Map([
-        [Blockchain.ETHEREUM, "https://eth.llamarpc.com"],
-        [Blockchain.POLYGON, "https://polygon.llamarpc.com"],
-        [Blockchain.BSC, "BNB_BSC"],
-        [Blockchain.OPTIMISM, "https://optimism.llamarpc.com"],
-        [Blockchain.ARBITRUM, "https://arbitrum.llamarpc.com"],
-        [Blockchain.GOERLI, "https://ethereum-goerli.publicnode.com"],
-        [Blockchain.SEPOLIA, "https://ethereum-sepolia.publicnode.com"],
-        [Blockchain.MUMBAI, "https://rpc-mumbai.maticvigil.com"],
-        [Blockchain.ZKATANA, "https://rpc.startale.com/zkatana"],
+export function getUrlProviderByBlockchain(chain: BlockchainIncludingTestnet) {
+    return new Map<BlockchainIncludingTestnet, string>([
+        ["ethereum", "https://eth.llamarpc.com"],
+        ["polygon", "https://polygon.llamarpc.com"],
+        ["bsc", "BNB_BSC"],
```

Comment:
I wonder if we need to throw error or provide `null` here so that it's obvious what the issue is when we're trying to use an RPC we don't have